lib/help.js

```diff
@@ -240,11 +240,11 @@ class Help {
     if (cmd._aliases[0]) {
       cmdName = cmdName + '|' + cmd._aliases[0];
     }
-    let parentCmdNames = '';
-    for (let parentCmd = cmd.parent; parentCmd; parentCmd = parentCmd.parent) {
-      parentCmdNames = parentCmd.name() + ' ' + parentCmdNames;
-    }
-    return parentCmdNames + cmdName + ' ' + cmd.usage();
+    let ancestorCmdNames = '';
+    cmd._getCommandAndAncestors().slice(1).forEach((ancestorCmd) => {
```

Comment:
No. Having to use the slice makes this too subtle for my liking.
Ok with the active variable changing to ancestor though.